spec/features/user_receives_email_for_new_review_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature "user receives an email when a new
+review is posted on a book they created", %{
+As a user
+I want to receive an email
+when somebody posts a review on a book I've submitted
+} do
+  # Acceptance Criteria
+  # [] As a user, I want to receive an email when
+  #    another user posts a review on a book I created
+
+  scenario "review a book" do
+    ActionMailer::Base.deliveries.clear
+
+    user = FactoryGirl.create(:user)
+    genre = FactoryGirl.create(:genre)
+    author = FactoryGirl.create(:author)
+    book = FactoryGirl.create(
+      :book,
+      author_id: author.id,
+      user_id: user.id,
+      genre_id: genre.id)
+
```

Comment:
For next time, you could consider moving this outside the scenario (e.g., in a `let` block)